This patch was introduced by me, to allow the access of other users to root-mounted captive-ntfs volumes.

--- captive-1.1.5/src/client/lufs/mount.captive.in.orig	2004-12-05 21:59:19.561715008 +0100
+++ captive-1.1.5/src/client/lufs/mount.captive.in	2004-12-05 22:00:49.549034856 +0100
@@ -127,8 +127,10 @@
 
 warn "$0: '-n' not supported - ignored" if $nomtab;
 
+my $fuseopts="-oallow_other,fsname=$image";
+
 # Use "'$oo'" to permit another expansion by bash(1) during lufsmnt(8) execution.
-my @argv=($lufsd_bin,$oo,$dir,"-s");
+my @argv=($lufsd_bin,$oo,$dir,"-s",$fuseopts);
 
 print STDERR "$0: @argv\n" if $verbose;
 exec $lufsd_bin @argv;
